scripts/services/check-svg.js

```diff
@@ -0,0 +1,74 @@
+const { DOMParser } = require('xmldom');
+
+/**
+ * Parses an SVG string and checks its syntax.
+ *
+ * @param {string} serviceName - The name of the service associated with the SVG.
+ * @param {string} SVGString - The SVG string to parse.
+ * @throws {Error} If there is an SVG syntax error, an error is thrown.
+ */
+const checkSVG = (serviceName, SVGString) => {
+    /**
+     * Parses the SVG string into an SVG DOM object.
+     *
+     * @param {string} SVGTag - The SVG string to parse.
+     * @returns {Document} The parsed SVG DOM object.
+     * @throws {Error} If there is an SVG syntax error, an error is thrown.
+     */
+    const parseSVG = (SVGTag) => {
+        const parser = new DOMParser({ locator: {}, errorHandler: {} });
+        try {
+            const SVGObject = parser.parseFromString(SVGTag);
+            return SVGObject;
+        } catch (error) {
+            throw new Error(`${serviceName} : SVG syntax error`);
+        }
+    };
+
+    const svgNode = parseSVG(SVGString, serviceName);
+
+    if (!svgNode.childNodes[0].nodeName || svgNode.childNodes[0].nodeName !== 'svg') {
+        throw new Error(`${serviceName} : Parsed SVG object is undefined`);
+    }
+
+    const svgElement = svgNode.documentElement;
+
+    /**
+     * Checks if the SVG tag contains `width` and `height` attributes.
+     *
+     * @throws {Error} If `width` or `height` attributes are present, an error is thrown.
+     */
+    const checkWidthHeightAttributes = (svgItem) => {
+        if (svgItem.hasAttribute('height') || svgItem.hasAttribute('width')) {
+            throw new Error(`${serviceName} : Svg tag must not contain \`width\` and \`height\` attributes`);
```

Comment:
tbh, not sure about backticks in the error message, i usually use `'`

```suggestion
            throw new Error(`${serviceName} : Svg tag cannot have a 'height' or 'width' attribute`);
```